tests/integration/devfile/cmd_devfile_watch_test.go

```diff
@@ -322,6 +195,81 @@ var _ = Describe("odo devfile watch command tests", func() {
 			}, 180, 10).Should(Equal(true))
 
 		})
+
+	})
+
+	When("a component is created and pushed", func() {
```

Comment:
We should try to avoid using the same description for different When's, but this one is the same as in line 55. 
Can you explain in the `When` the peculiarity (it seems to be about using the --build-command and --run-command)

Reply:
I updated the statement to reflect the use of --build-command and --run-command flags